scripts/vue_or_react.sh

```diff
@@ -2,8 +2,8 @@
 
 # cookiecutter tn-spa-bootstrapper --config-file tn-spa-bootstrapper/cookiecutter/vue_template.yaml --no-input
 git fetch origin main
-react_count=$(git diff --name-only origin/main -- | grep "/clients/web/react" | wc -l)
-vue_count=$(git diff --name-only origin/main -- | grep "/clients/web/vue" | wc -l)
+react_count=$(git diff --name-only origin/main -- | grep "/clients/web/react/" | wc -l)
+vue_count=$(git diff --name-only origin/main -- | grep "/clients/web/vue/" | wc -l)
```

Comment:
adding the slash here will break things.
One solution is the make this `vue3` with the slash included.

Reply:
So we should rename the vue folder to vue3?
And this would end up being `[...]/web/vue3/` ?